reimbursement.md

```diff
@@ -282,6 +282,96 @@ Find the reimbursement status in the transaction `reimbursement.status` property
 
 - `failed`: scheme request failed and `transaction.reimbursement.error` object is created. Retry is possible. [See error list for more information](https://fidel.uk/docs/reimbursement#errors).
 
+## Reimbursement by card
+
+At this point reimbursements could only be performed for a specific transaction. There are other scenarios when you just want to reimburse a card, without a transaction. In this case, you'd need to find a transaction (that belongs to the card) and is  eligible for reimbursement manually, then reimburse. Depending on your use case, this can be a time consuming and tedious process.
```

Comment:
I would rephrase the first two paragraphs. Users usually don't read documentation pages from the beginning to the end, so this page is actually a compilation of smaller texts that the reader will read when they think it's a solution for their question (and they usually skip the rest).

Suggestion: 
"In some cases, you may want to reimburse a card (and not a specific transaction). However, as card networks allow reimbursements only on specific transactions, the reimbursement request needs to be based on a transaction. The `/cards/{cardId}/reimbursement` endpoint helps you to find such a transaction for this case. You can simply define the `cardId`, and Fidel API will search for the most suitable transaction, and apply the reimbursement on that one.